pkgs/development/libraries/libuv/default.nix

```diff
@@ -38,7 +38,7 @@ stdenv.mkDerivation rec {
   # These should be turned back on, but see https://github.com/NixOS/nixpkgs/issues/23651
   # For now the tests are just breaking large swaths of the nixpkgs binary cache for Darwin,
   # and I'd rather have everything else work at all than have stronger assurance here.
-  doCheck = !stdenv.isDarwin;
+  doCheck = !stdenv.isDarwin && stdenv.buildPlatform == stdenv.hostPlatform;
```

Comment:
We shouldn't propagate this everywhere. Stdenv should just disable doCheck when cross compiling.